### PR TITLE
Regression: Removing user from team doesn't remove them from the team's room.

### PR DESCRIPTION
**Pull request by KevLehman**
## Proposed changes (including videos or screenshots)

- Remove subscription when calling `teams.removeMembers`

## Further comments

- It looks like there are some FE Eslint issues, that may cause pipelines to fail
- I had to use the Subscription model (not raw) because FE was not reading the changes when doing the delete using the raw model.

### CODE REVIEW

File: server/services/team/service.ts [hunk 1]
```diff
@@ -32,6 +33,7 @@ export class TeamService extends ServiceClass implements ITeamService {
 	private MessagesModel: MessagesRaw;
 
 	// TODO not sure we should have the collection here or call the Room service for getting Room data
+	// What's the difference between this one and the one on L25?
```
Comment (pierre-lehnen-rc) [inline server/services/team/service.ts:36]: It's duplicated, you can remove this one.

File: server/services/team/service.ts [hunk 2]
```diff
@@ -566,13 +568,9 @@ export class TeamService extends ServiceClass implements ITeamService {
...
 
 	async removeMembers(teamId: string, teamName: string, members: Array<ITeamMemberParams>): Promise<void> {
-		if (!teamId) {
-			const teamIdName = await this.TeamModel.findOneByName(teamName, { projection: { _id: 1 } });
-			if (!teamIdName) {
-				throw new Error('team-does-not-exist');
-			}
-
-			teamId = teamIdName._id;
+		const team = await this.TeamModel.findOneByIdOrName(teamId || teamName, { projection: { _id: 1, roomId: 1 } });
```
Comment (pierre-lehnen-rc) [inline server/services/team/service.ts:571]: This isn't safe. It's better to search for id when we get an id and search for name when we get a name.

**Review by pierre-lehnen-rc — CHANGES_REQUESTED** [review]
Check both comments on the code.

File: server/services/team/service.ts (hunk 2 above)
Comment (KevLehman) [reply server/services/team/service.ts:571]: Just curious, why do you say it's not safe? 👀

File: server/services/team/service.ts (hunk 1 above)
Comment (KevLehman) [reply server/services/team/service.ts:36]: Nice, thanks

File: server/services/team/service.ts (hunk 2 above)
Comment (pierre-lehnen-rc) [reply server/services/team/service.ts]: there can be a room with the name of another room's id (unlikely, but possible). Then the query could pick the wrong room here.

**Review by pierre-lehnen-rc — APPROVED** [review]

**Review by pierre-lehnen-rc — DISMISSED** [review]

**Review by pierre-lehnen-rc — APPROVED** [review]

**Pull request merged**